### PR TITLE
changed language settings on different positions

### PR DESCRIPTION
**Pull request by DanielNowak98**
Changed Language Linkings on different Positions.

### CODE REVIEW

File: src/languages.hpp [hunk 1]
```diff
@@ -33,6 +40,10 @@ enum class Key {
   Start,
   Stop,
   Continue,
+  Cancel,
```
Comment (R-Abbasi) [inline src/languages.hpp:43]: _Cancel_ was not added because by default _Back_ was used instead of that.

File: src/languages.hpp [hunk 2]
```diff
@@ -73,14 +84,23 @@ enum class Key {
...
     {Key::Settings, "Settings"},
     {Key::Reset, "Reset"},
     {Key::Reset_q, "Reset?"},
     {Key::Measure_not_saved, "The measurement was not saved!\n"
                              "Would you like to save it before deleting it?\n"},
+    {Key::Restore, "Restore"},
+    {Key::Devicelist, "Devicelist"},
+    {Key::RecordData, "Record Data"},
+    {Key::Recordingdata, "Recording the data"},
+    {Key::Save, "Save"},
```
Comment (R-Abbasi) [inline src/languages.hpp:99]: _Save_ exists already, I guess.

File: src/main.cpp [hunk 3]
```diff
@@ -169,27 +168,31 @@ int main() {
     ImGui::Dummy({0.f, windowSize.y * .01f});
     ImGui::PushStyleVar(ImGuiStyleVar_ChildBorderSize, windowSize.x * .011f);
     ImGui::PushStyleColor(ImGuiCol_Border, {0.3f, 0.4f, 0.7f, 0.6f});
-    ImGui::BeginChild("Record Data", {0.f, windowSize.y * 0.5f},
+    ImGui::BeginChild(appLanguage[Key::RecordData], {0.f, windowSize.y * 0.5f},
```
Comment (R-Abbasi) [inline src/main.cpp:171]: `ImGui::BeginChild`'s _id_ is not normally visible, so can't be sure how modifying it helps.

File: src/languages.hpp (hunk 1 above)
Comment (DanielNowak98) [reply src/languages.hpp:43]: Ahh okay, thank you!

File: src/handler.cpp [hunk 4]
```diff
@@ -1,7 +1,8 @@
-#include <set>
-#include <implot.h>
 #include "handler.hpp"
 #include "get_from_github.hpp"
+#include "languages.hpp"
+#include <implot.h>
+#include <set>
```
[the remark this answers is not in the document]
Comment (DanielNowak98) [reply src/handler.cpp:5]: I think this was done by the clang formatter.

File: src/main.cpp (hunk 3 above)
Comment (AKMaily) [reply src/main.cpp:171]: In the child window for the plot the title is visible and is currently only in english @R-Abbasi

Comment (R-Abbasi) [reply src/main.cpp:171]: Do you mean [this BeginChild](https://github.com/skunkforce/OmniView/blob/master/src/main.cpp#L172), please?

Comment (R-Abbasi) [reply src/main.cpp:171]: As well as, it's not a title but an ID. The title is set by `addPlots("Recording the data" ...`.

Comment (AKMaily) [reply src/main.cpp:171]: Okay got it. So the language has to be changed directly in the addPlots function and not in BeginChild. @DanielNowak98

**Pull request closed without merging**